drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -2566,7 +2566,7 @@ Vector<uint8_t> RenderingDeviceVulkan::_texture_get_data_from_image(Texture *tex
 					for (uint32_t y = 0; y < height; y++) {
 						const uint8_t *rptr = slice_read_ptr + y * layout.rowPitch;
 						uint8_t *wptr = write_ptr + y * pixel_size * width;
-						copymem(wptr, rptr, pixel_size * width);
+						copymem(wptr, rptr, (long) pixel_size * width);
```

Comment:
If I'm understanding the other comments on this PR correctly, this line should be like this:
```suggestion
						copymem(wptr, rptr, (long)pixel_size * width);
```

Reply:
> Or does it affect the result of `pixel_size * width`?)

It may affect if those two values are very big: the values may overflow (see an explanation from [LGTM](https://lgtm.com/rules/2157860313/) for more information.

> Don't know if it needs to be `(uint64_t)` since the checks seemed to of passed?

This is fixed now, thanks. 🙂